spyder_kernels/utils/tests/test_nsview.py

```diff
@@ -18,12 +18,13 @@
 import pandas as pd
 import pytest
 import xarray as xr
+import PIL
```

Comment:
Please add `pillow` to our requirements for our tests in our `setup.py` and `requirements/tests.txt`.